### DISCUSSION TITLE
Better contact?

### DISCUSSION THREAD

**Discussion by PantherPK** [RyseInventory/RyseInventory#56] [Q&A] [answered]
Can i know if there is any discord server for RyseInventory community or smth? or will be any in furture?

**Comment by Rysefoxx** [comment, marked as answer]
Is in development. Unfortunately I have less time at the moment.
I will publish more information when I am ready. Planned is a Discord Bot that I'm still programming, where you can report wishes/bugs. Furthermore, that you can also open tickets there.
